src/main/resources/lang/lang_de_DE.properties

```diff
@@ -49,9 +49,9 @@ CmdFactionFlagSet=setzen
 CmdFactionBypass=umgehen
 CmdFactionChat=chat
 CmdFactionChatHistory=geschichte
-CmdFactionClaimFill=ansprucherf�llen
+CmdFactionClaimFill=ansprucherf�llen
```

Comment:
The German and French translation files seem to have been messed up a bit. This needs to be resolved before this PR is merged into develop.

Reply:
Yep, accents need to be sorted out